### PR TITLE
Plugins Catalog: Fix a11y issues

### LINKED ISSUE #37933: Plugins Catalog: check a11y with https://www.deque.com/axe/

### PR DESCRIPTION
**Pull request by jackw**
**What this PR does / why we need it**:
This PR addresses accessibility issues raised by the axe DevTools extension.

| Before | After |
| --- | --- |
| Screenshot 2021-09-20 at 12 15 13 | Screenshot 2021-09-20 at 12 17 18 |

**Which issue(s) this PR fixes**:

Fixes #37933

**Special notes for your reviewer**:
There are two remaining "Needs Review" issues in the PluginDetails page that are due to the usage of `aria-label` in the `<Tab />` component.

Screenshot 2021-09-20 at 12 19 59

### CODE REVIEW

File: public/app/features/plugins/admin/components/PluginListCard.tsx [hunk 1]
```diff
@@ -26,10 +26,10 @@ export function PluginListCard({ plugin, pathName }: PluginListCardProps) {
             className={styles.image}
             height={LOGO_SIZE}
           />
-          <h3 className={styles.name}>{plugin.name}</h3>
+          <h2 className={styles.name}>{plugin.name}</h2>
           {plugin.type && (
-            <div className={styles.icon}>
-              <Icon name={IconName[plugin.type]} aria-label={`${plugin.type} plugin icon`} />
+            <div className={styles.icon} data-testid={`${plugin.type} plugin icon`}>
+              <Icon name={IconName[plugin.type]} />
```
Comment (leventebalogh) [inline public/app/features/plugins/admin/components/PluginListCard.tsx:32]: **Question:** why did it not like the `area-label` on this one?

**Review by leventebalogh — APPROVED** [review]
LGTM 👍

File: public/app/features/plugins/admin/components/PluginListCard.tsx (hunk 1 above)
Comment (jackw) [reply public/app/features/plugins/admin/components/PluginListCard.tsx:32]: The `<Icon />` adds the `aria-label` to a `<div />` which isn't a valid element for the attribute. https://www.tpgi.com/short-note-on-aria-label-aria-labelledby-and-aria-describedby/

Comment (leventebalogh) [reply public/app/features/plugins/admin/components/PluginListCard.tsx:32]: Thanks mate 👍

**Pull request merged**